src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue

```diff
@@ -127,6 +127,18 @@ export default {
           }
         }
 
+        // check if this is a text token following an attribute token
+        // so we can insert a newline here and split each attribute onto its
+        // own line
+        //
+        // we want to avoid doing this when the attribute is encountered
+        // in a param clause for attributes like `@escaping`
```

Comment:
Looks great! Can we add a test for this scenario?

Reply:
0cc2a2476858c45cd0c376444cc12c8edbdeaf6c